core/java/iesi-core/src/main/java/io/metadew/iesi/metadata/configuration/execution/ExecutionRequestConfiguration.java

```diff
@@ -167,8 +169,9 @@ private ExecutionRequestConfiguration() {
                             ExecutionRequestStatus.valueOf(cachedRowSet.getString("ST_NM")),
                             ScriptExecutionRequestConfiguration.getInstance().getByExecutionRequest(new ExecutionRequestKey(cachedRowSet.getString("REQUEST_ID"))),
                             ExecutionRequestLabelConfiguration.getInstance().getByExecutionRequest(new ExecutionRequestKey(cachedRowSet.getString("REQUEST_ID"))),
-                            cachedRowSet.getString("SPACE_NM"),
-                            cachedRowSet.getString("USER_NM"), cachedRowSet.getString("USER_PASSWORD")));
+                            cachedRowSet.getString("USER_ID"),
+                            cachedRowSet.getString("USERNAME")
+                            ));
```

Comment:
could you remove the comments on line 190-... and also the other blocks of comments